clang/lib/ASTMatchers/Dynamic/Registry.cpp

```diff
@@ -125,6 +125,7 @@ RegistryMaps::RegistryMaps() {
   REGISTER_MATCHER_OVERLOAD(equals);
 
   REGISTER_REGEX_MATCHER(isExpansionInFileMatching);
+  REGISTER_REGEX_MATCHER(matchesString);
   REGISTER_REGEX_MATCHER(matchesName);
   REGISTER_REGEX_MATCHER(matchesSelector);
```

Comment:
```suggestion
  REGISTER_REGEX_MATCHER(matchesName);
  REGISTER_REGEX_MATCHER(matchesSelector);
  REGISTER_REGEX_MATCHER(matchesString);
```
We try to keep these in alphabetical order.

Reply:
thanks,fixed in https://github.com/llvm/llvm-project/pull/102152/commits/df10be46105dfa73278c7a3bd78a02ce21a93f97